src/container/__integ__/awsui-applayout-sticky.test.ts

```diff
@@ -16,7 +16,7 @@ const stickyToggleSelector = createWrapper().findFlashbar().findItems().get(1).f
 const headerSelector = '#b #h';
 const scrollableDivSelector = '#scrollable-div';
 
-const CONTAINER_ROOT_BORDER = 1;
+const CONTAINER_ROOT_BORDER = 0;
```

Comment:
Sane as other comments of this could be removed.

Reply:
Repasting same response for visibility - Originally was there because we had [this code](https://github.com/cloudscape-design/components/blob/40a405bd591436179a2e1e9a4287664bc7f6582c/src/container/use-sticky-header.ts#L25-L33) that checks for the container and header borders and accounts for it in sticky header. Removed both in next revision because borders no longer contribute to container height.